### PR TITLE
Formatting Dates for download CSV at a point

### PR DESCRIPTION
**Pull request by pmbohm**

### CODE REVIEW

File: src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/TextCsvConverter.java [hunk 1]
```diff
@@ -154,8 +164,23 @@ private void writeCsvLine(PrintStream out, Map<Variable, NetcdfReader> variableR
         Iterator<Variable> variableReaderIterator = variableReaders.keySet().iterator();
 
         while (variableReaderIterator.hasNext()) {
-            Variable variable = variableReaderIterator.next();
-            out.print(getDisplayValue(variable, variableReaders.get(variable), indexTuple));
+            Variable ncVariable = variableReaderIterator.next();
+
+            Attribute axis = ncVariable.findAttribute("axis");
+            if (axis != null && axis.getStringValue().equals("T")) {
```
Comment (jonescc) [inline src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/TextCsvConverter.java]: Sounds like something that should go into getDisplayValue?

File: src/extension/wps/src/test/resources/expected_timeseries.csv [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
-24252.6875,-31.774258,115.06208,72.611725,0.15903877,-0.6810493,0.061944697,0.06507293,6,5,0,0
-24252.729166666668,-31.774258,115.06208,72.611725,,,,,,,,
-24252.770833333332,-31.774258,115.06208,72.611725,,,,,,,,
-24252.8125,-31.774258,115.06208,72.611725,0.16106902,-0.6671398,0.03751583,0.04652556,6,6,0,0
-24252.854166666668,-31.774258,115.06208,72.611725,0.13536559,-0.6338494,0.034652464,0.046642102,6,6,0,0
-24252.895833333332,-31.774258,115.06208,72.611725,0.16587818,-0.6182532,0.027516443,0.04727751,6,6,0,0
-24252.9375,-31.774258,115.06208,72.611725,0.15575635,-0.62707496,0.02892067,0.049721044,6,6,0,0
+TIME (UTC),LATITUDE (degrees_north),LONGITUDE (degrees_east),GDOP (Degrees),UCUR (m s-1),VCUR (m s-1),UCUR_sd (m s-1),VCUR_sd (m s-1),NOBS1 (1),NOBS2 (1),UCUR_quality_control,VCUR_quality_control
+2016-05-26T00:30:00Z,-31.774258,115.06208,72.611725,0.35732692,-0.2912578,0.043182936,0.067261055,6,6,0,0
+2016-05-26T01:30:00Z,-31.774258,115.06208,72.611725,0.30322826,-0.32465956,0.035217717,0.053472657,6,6,0,0
```
Comment (jonescc) [inline src/extension/wps/src/test/resources/expected_timeseries.csv]: Sweet!

**Comment by pmbohm** [comment]
@jonescc Review changes done. Thanks

**Comment by jonescc** [comment]
tested on vagrant and doesn't appear to work for GSLA.   Doesn't use axis: T.   This works with ankhit's time variable detection - we should really use that (i.e. the netcdf library)

**Comment by bhasin85** [comment]
@jonescc  @julian1  @pmbohm  @anguss00 @lwgordonimos  PR ready to be reviewed. :+1:

File: src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/TextCsvConverter.java [hunk 3]
```diff
@@ -119,8 +120,7 @@ private void writeHeaderLine(PrintStream out, List<Variable> variables) {
             String shortName = ncVariable.getShortName();
             out.print(shortName);
 
-            Attribute axis = ncVariable.findAttribute("axis");
-            if (axis != null && axis.getStringValue().equals("T")) {
+            if (shortName.equals(module.getTime().getShortName())) {
```
Comment (pmbohm) [inline src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/TextCsvConverter.java]: Agh.. The missing link in place. nice one @bhasin85 !!

**Pull request merged**